src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=568469823&format=interactive",
+  sheets_gid="2090334936",
+  sql_file="request_type_distribution_2021.sql"
+) }}
+
+Images again make up the largest number of requests, though JavaScript is closing in as the gap has narrowed slightly in the last year.
+
+### File formats
+
+{{ figure_markup(
+  image="response-distribution-format.png",
+  caption="Distribution of image sizes by format.",
+  description="Box plot of the distribution of image sizes by format: gif, ico, jpg, png, svg, and webp. JPG sticks out as having the highest distribution with a 90th percentile exceeding 190 KB per image, up from the previous year. WebP has overtaken PNG as second highest at about 93 KB at the 90th percentile. PNG is now third with smaller 90th percentile than WebP, its 75th percentile is higher. gif, ico, and svg all have relatively tiny distributions.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=1911661099&format=interactive",
+  sheets_gid="38001790",
+  sql_file="requests_format_distribution.sql"
+) }}
+
+We know that images responsible for a large percentage of web page weight. The above graphic shows the top sources of image weight and the weight distribution. Top 3: JPG, WebP and PNG. So not only is the JPG the most popular image format, it also tends to be the largest by size as well - even larger than a lossless format like the PNG. Likely, the rise in popularity for WebP is due to all major browsers finally supporting the file format. PNG remains popular for use cases such as icons and logos.
+
+### Image bytes
+
+{{ figure_markup(
+  image="response-distribution-images.png",
+  caption="Distribution of image response sizes per page.",
+  description="Bar chart showing the distribution of image bytes per page. Desktop pages tend to load more image bytes per page throughout the distribution. The 10, 25, 50, 75, and 90th percentiles for mobile pages are: 62, 257, 877, 2,324, and 4,992 KB of images per page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=1440589973&format=interactive",
+  sheets_gid="895329225",
+  sql_file="request_type_distribution_2021.sql"
+) }}
+
+Looking at total image bytes shows us that this metric has remained virtually unchanged from the previous year. This is likely due more and more images being served by content distribution networks (CDN), which apply strong optimizations to images as they are uploaded to their servers for use on future requests.
```

Comment:
We don't see the previous year here so this does not "show" us that. This chart only shows desktop and mobile at different percentiles for this year.